### PR TITLE
Add Custom WebViewActivity instructions

### PR DESCRIPTION
**Pull request by bryanlogan**
# Pull Request/Issue Resolution

Closes #N/A

#### Is this change associated with a Braze feature/product release?
- [ x ] Yes (4.0.0 Nov 18, 2019)
- [ ] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ x ] Check that all links work.
- [ x ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the chart below to tag the appropriate reviewer.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTtzHpcihaXTTYD85LoKIvYBvpCQFLr8n0BDKRDRAMEz_DnZdHJJINKL24r4JXkRUui24pl_DVxbu2T/pub)
- [Technical Documentation Style Guide](https://docs.google.com/document/d/e/2PACX-1vTtzHpcihaXTTYD85LoKIvYBvpCQFLr8n0BDKRDRAMEz_DnZdHJJINKL24r4JXkRUui24pl_DVxbu2T/pub#h.wstt3flbts5k)
- [Styling Test Page](https://github.com/braze-inc/braze-docs/blob/develop/_docs/_home/styling_test_page.md)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [x] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [x] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then `Docs`, this can take up to 5 minutes to load. A `502` error means the deployment is still loading, refresh the page, and try again.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @timothy-kim or @KellieHawks for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@Timothy-Kim</td>
    <td>Application Infrastructure<br>Data Infrastructure</td>
  </tr>
  <tr>
    <td>@kelliehawks</td>
    <td>Intelligence<br>Product Partnerships<br>SMS<br>Internal Tools</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Reporting<br>Ingestion<br>Platforms and Channels<br>SMB</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Messaging and Automation<br>Dashboard Infrastructure<br>Email</td>
  </tr>
</table>
</details>

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/android/advanced_use_cases/deep_linking.md [hunk 1]
```diff
@@ -179,3 +179,55 @@ Then, as you set up your push notification campaign (either through the [dashboa
 [udl-1]: https://appboy.github.io/appboy-android-sdk/javadocs/com/appboy/ui/actions/UriAction.html
 [udl-2]: https://appboy.github.io/appboy-android-sdk/javadocs/com/appboy/ui/AppboyNavigator.html#setAppboyNavigator-com.appboy.IAppboyNavigator-
 [udl-3]: https://appboy.github.io/appboy-android-sdk/javadocs/com/appboy/IAppboyNavigator.html
+
+## Custom WebView Activity {#Custom_Webview_Activity}
+
+By default, deeplinks to websites are are handled by [`BrazeWebViewActivity`][udl-4]. To change this:
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/android/advanced_use_cases/deep_linking.md:185]:
```suggestion
By default, when website deeplinks are opened inside the app by Braze, they are handled by [`BrazeWebViewActivity`][udl-4]. To change this:
```

File: _docs/_developer_guide/platform_integration_guides/android/advanced_use_cases/deep_linking.md [hunk 2]
```diff
@@ -179,3 +179,55 @@ Then, as you set up your push notification campaign (either through the [dashboa
...
+
+1. Create a new Activity that handles the target URL from `Intent.getExtras()` with the key `com.appboy.Constants.APPBOY_WEBVIEW_URL_EXTRA`. See [`BrazeWebViewActivity.java`][udl-8] for an example.
+2. Add that activity to `AndroidManifest.xml` and set exported to false.
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/android/advanced_use_cases/deep_linking.md:188]:
```suggestion
2. Add that activity to `AndroidManifest.xml` and set `exported` to `false`.
```

File: _docs/_developer_guide/platform_integration_guides/android/advanced_use_cases/deep_linking.md [hunk 3]
```diff
@@ -179,3 +179,55 @@ Then, as you set up your push notification campaign (either through the [dashboa
...
+
+    ```xml
+    <activity
+        android:name=".MyCustomWebViewActivity"
+        android:exported="false" />
+    ```
+
+3. Use a [builder object][udl-6] that is then built and passed to [Braze.configure()][udl-5] in [`Application.onCreate()`][udl-7]
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/android/advanced_use_cases/deep_linking.md:196]:
```suggestion
3. Set your custom Activity in a `BrazeConfig` [builder object][udl-6]. Build the builder and pass it to [Braze.configure()][udl-5] in your [`Application.onCreate()`][udl-7]
```

nit to help avoid passive voice

**Review by Bucimis — APPROVED** [review]

**Review by KellieHawks — APPROVED** [review]
Looks great! Thanks for committing to the Braze repo!

**Review by chshapiro — APPROVED** [review]

**Pull request merged**